core/node/house_keeper/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "zksync_house_keeper"
+version = "0.1.0"
+edition.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+keywords.workspace = true
+categories.workspace = true
+
+[dependencies]
+vise.workspace = true
+zksync_dal.workspace = true
+zksync_shared_metrics.workspace = true
+zksync_utils.workspace = true
+zksync_db_connection.workspace = true
```

Comment:
I don't think this dependency is necessary; AFAICT, house keeper only uses `ConnectionPool`, which is re-exported from `zksync_dal`.

Reply:
>which is re-exported from zksync_dal.

wdym?
from what I see its not - https://github.com/matter-labs/zksync-era/blob/9948187aa12bcd99ef512ec8cf322fefe2cd745b/core/lib/db_connection/src/connection_pool.rs#L249

Reply:
`ConnectionPool` is defined in `zksync_db_connection`, yes, but it's re-exported from `zksync_dal` as well:

https://github.com/matter-labs/zksync-era/blob/9948187aa12bcd99ef512ec8cf322fefe2cd745b/core/lib/dal/src/lib.rs#L10

Reply:
ah i see, ok